pylatex/tikz.py

```diff
@@ -98,3 +98,22 @@ def dumps(self):
         super().dumps()
 
         return string
+
+
+class Frame(Environment):
```

Comment:
Why is this in the Tikz module. I think it would be best to create a new module called frames.
Where all different types can be added, like the fbox one and this one. 

This should also just be called mdframed, to make clear that is the frame type that is used.